pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -527,15 +534,6 @@ func (c *Reconciler) reconcile(ctx context.Context, pr *v1alpha1.PipelineRun) er
 		}
 	}
 
-	// If the pipelinerun is cancelled, cancel tasks and update status
-	if pr.IsCancelled() {
-		before := pr.Status.GetCondition(apis.ConditionSucceeded)
-		err := cancelPipelineRun(c.Logger, pr, pipelineState, c.PipelineClientSet)
-		after := pr.Status.GetCondition(apis.ConditionSucceeded)
-		reconciler.EmitEvent(c.Recorder, before, after, pr)
-		return err
```

Comment:
in this version, the `reconcile` function would return at this point if cancelled, but looks like that is no longer the case - want to double check we want to keep executing `reconcile` in this case

Reply:
In the new version we run even less reconcile than before. We hit this condition in the `Reconcile` and never enter `reconcile`.

Reply:
you're right! i was misreading, i thought that the reconcile still happened